Domain/Constants/SeatTypes.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Domain.Constants
+{
+    public enum SeatTypes : byte
+    {
+        LongTerm = 0,
```

Comment:
it's better to start with 1, 0 is default and usually used for error values, or not set